Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -2759,6 +2785,9 @@ - (void)_didStopDeferringGeometryUpdates
 {
     [self _scheduleVisibleContentRectUpdate];
 
+    // This should do at least all the things that had been skipped in `-[WKWebView _frameOrBoundsMayHaveChanged]`,
```

Comment:
Can we enforce that in code somehow?

Reply:
I factored out as much common code as possible, but the rest of it gets intertwined with other properties being set, and I didn't want to change the order of that, so I left as is.